src/App.tsx

```diff
@@ -1,24 +1,246 @@
-/* eslint-disable max-len */
-/* eslint-disable jsx-a11y/control-has-associated-label */
-import React from 'react';
+import React, {
+  useCallback,
+  useEffect,
+  useMemo,
+  useState,
+} from 'react';
 import { UserWarning } from './UserWarning';
+import { Todo } from './types/Todo';
+import {
+  getTodos,
+  addTodo,
+  deleteTodo,
+  completeTodo,
+  editTodo,
+} from './api/todos';
+import { Filter } from './types/Filter';
+import { ErrorTypes } from './types/ErrorTypes';
+import {
+  Header, TodosList, Footer, ErrorNotifications,
+} from './components';
 
-const USER_ID = 0;
+const USER_ID = 6335;
 
 export const App: React.FC = () => {
+  const [allTodos, setAllTodos] = useState<Todo[]>([]);
+  const [error, setError] = useState<ErrorTypes | null>(null);
+  const [filter, setFilter] = useState<Filter>(Filter.All);
+  const [title, setTitle] = useState<string>('');
+  const [isAdding, setIsAdding] = useState<boolean>(false);
+  const [tempTodo, setTempTodo] = useState<Todo | null>(null);
+  const [selectedTodoIds, setSelectedTodoIds] = useState<number[]>([]);
+  const [isEditing, setIsEditing] = useState<boolean>(false);
+
+  const clearError = () => setError(null);
+
+  useEffect(() => {
+    getTodos(USER_ID)
+      .then(setAllTodos)
+      .catch(() => {
+        setError(ErrorTypes.Load);
+        setTimeout(clearError, 3000);
+      });
+  }, []);
```

Comment:
I see you can move more duplicated code inside function

```suggestion
  const pushError = (error: ErrorTypes, time = 3000) => {
    setError(error);
    setTimeout(() => setError(null), time);
  };

  useEffect(() => {
    getTodos(USER_ID)
      .then(setAllTodos)
      .catch(() => pushError(ErrorTypes.SomeError));
  }, []);
```